kf_lib_data_ingest/config.py

```diff
@@ -2,6 +2,7 @@
 import logging
 
 ROOT_DIR = os.path.dirname(__file__)
+TEMPLATES_DIR = os.path.join(os.path.dirname(ROOT_DIR), 'templates')
```

Comment:
dirname of dirname?

Reply:
Yea, I had to do this so that there are no conflicts with our tests/conftest.py and the templates/my_study/tests/conftest.py.

Reply:
oh right